kernel/ramfs.c

```diff
@@ -2964,6 +2966,34 @@ int myst_create_virtual_file(
     return ret;
 }
 
+int myst_read_virtual_file(myst_file_t* file, void* buf, size_t buf_size)
```

Comment:
Is this read function generalizable? It seems like its suitable for stateful virtual files, where reads consume from `file->vbuf`.

Reply:
As this is specific to stateful virtual files, can we qualify the function name to suggest that?
Maybe `myst_read_stateful_virtual_file`.

Reply:
Renamed.